json/es.json

```diff
@@ -1,71 +1,71 @@
     "Are you sure you want to delete your account? Once your account is deleted, all of its resources and data will be permanently deleted. Please enter your password to confirm you would like to permanently delete your account.": "¿Está seguro que desea eliminar su cuenta? Una vez que se elimine su cuenta, todos sus recursos y datos se eliminarán de forma permanente. Ingrese su contraseña para confirmar que desea eliminar su cuenta de forma permanente.",
-    "Are you sure you want to detach the selected resources?": "Are you sure you want to detach the selected resources?",
-    "Are you sure you want to detach this resource?": "Are you sure you want to detach this resource?",
-    "Are you sure you want to force delete the selected resources?": "Are you sure you want to force delete the selected resources?",
-    "Are you sure you want to force delete this resource?": "Are you sure you want to force delete this resource?",
-    "Are you sure you want to restore the selected resources?": "Are you sure you want to restore the selected resources?",
-    "Are you sure you want to restore this resource?": "Are you sure you want to restore this resource?",
-    "Are you sure you want to run this action?": "Are you sure you want to run this action?",
+    "Are you sure you want to detach the selected resources?": "¿Está seguro que desea desvincular los recursos seleccionados?",
+    "Are you sure you want to detach this resource?": "¿Está seguro que desea desvincular este recurso?",
+    "Are you sure you want to force delete the selected resources?": "¿Está seguro que desea forzar la elimiación del recurso seleccionado?",
+    "Are you sure you want to force delete this resource?": "¿Está seguro que desea forzar la elimiación de este recurso?",
+    "Are you sure you want to restore the selected resources?": "¿Está seguro que desea restaurar el recurso seleccionado?",
+    "Are you sure you want to restore this resource?": "¿Está seguro que desea restaurar este recurso?",
+    "Are you sure you want to run this action?": "¿Está seguro que desea ejecutar esta acción?",
     "Are you sure you would like to delete this API token?": "¿Está seguro que desea eliminar este token API?",
     "Are you sure you would like to leave this team?": "¿Está seguro que le gustaría abandonar este equipo?",
     "Are you sure you would like to remove this person from the team?": "¿Está seguro que desea retirar a esta persona del equipo?",
     "Argentina": "Argentina",
     "Armenia": "Armenia",
     "Aruba": "Aruba",
-    "Attach": "Attach",
-    "Attach & Attach Another": "Attach & Attach Another",
-    "Attach :resource": "Attach :resource",
-    "August": "August",
+    "Attach": "Vincular",
+    "Attach & Attach Another": "Adjuntar & adjuntar otro",
+    "Attach :resource": "Adjuntar :resource",
+    "August": "Agosto",
     "Australia": "Australia",
```

Comment:
Without change, can you add these tags in script/excludes/es.php ?

Reply:
@caouecs, it's not obligatory. I brought the names of the countries into the global file of exceptions: [script/excludes/_all.php](https://github.com/Laravel-Lang/lang/blob/master/script/excludes/_all.php).

Reply:
I don't understand what you want me to do, I understand the English, but I am not familiarize with the project and I maybe be miss reading the logic of the project.

What is the `script/excludes/es.php` suppose to do?

(please don't tell me it excludes the translation from happening, that would defeat the porpoise of having the entry in the json file in the first place)

Reply:
You don't have to do anything.

This file does not exclude translation. It is needed to correctly form the todo.md file.

Reply:
ok